### DISCUSSION TITLE
Export to PDF: wrong fonts

### DISCUSSION THREAD

**Discussion by amlopezalonso** [freeplane/freeplane#358] [Help] [answered]
Hi all,

I'm experiencing the following issue: when exporting to PDF sans-serif fonts turn to serif in the exported file (Freeplane 1.7.10). Ticking text conversion to images and converting to some other formats (as XHTML and SVG) are not affected. As this did fully work some time in the past I suspect it must be due to some misconfiguration issue or maybe some missing packages on my side. Can someone give some guidance on what could be happening?

Regards,
Antonio

**Comment by dpolivaev** [comment, marked as answer]
The recent Freeplane version is 1.10.2.
Please check how it works and if needed submit a bug report including an example mind map and the PDF files generated by the old version where it was OK and by the latest version.

**Reply by amlopezalonso** [reply]
Thanks Dimitry. Updating to 1.10.2 fixed the issue but spawned another one: maps composed with the previous version show no edges (or rather they are invisible) when exported to PDF. Nevertheless I'll investigate this one further before submitting a bug report.

Thanks so much again.

Regards,
Antonio

**Reply by amlopezalonso** [reply]
Yet another similar issue has spawned in exported PDF: all node details texts have been turned into italics replacing Roboto font I chose, even removing bold characters wherever they were applied. I really don't know why as it seems all required fonts are installed and exporting to at least either XHTML and SVG shows no issues at all.

**Reply by dpolivaev** [reply]
Please don't mix different topics in the same discussion. If you see bugs submit bug reports under "issues" and use different issues for different findings. I do not always check and track things in the discussion forum

**Reply by amlopezalonso** [reply]
I apologize for that. I'll open a new report if I cannot find a config reason for the issue to happen.
